rasa/nlu/utils/__init__.py

```diff
@@ -33,6 +33,10 @@ def create_dir(dir_path: Text) -> None:
             raise
 
 
+def cmp_items(file):
+    return "_".join(file.split("_")[1:])
```

Comment:
why don't we sort files in alphabetic order?

Reply:
This is sorting the files in alphabetic order after removing the random string attached to the filename before first underscore.

Reply:
@tabergma

Reply:
@Ghostvv The prefix is added to the files here: https://github.com/RasaHQ/rasa/blob/master/rasa/data.py#L197. It is added, when the training files are copied to a temp dir.
So, in order to sort the files alphabetically the prefix needs to be removed. The method is doing exactly that.

Reply:
shall we do reordering in there? where we add the prefix?

Reply:
We cannot order in that method, as that method is just writing the files to disk. We should keep it here.

Reply:
This part has been moved to rasa/utils/io.py.